row_length: 10 
rows_count: 11 
rows_curve_budget: 0.78539816339 
row_segments: straight,curved 
row_segment_curved_radius_min: 4.0 
row_segment_curved_radius_max: 5.0 
ground_resolution: 0.15 
ground_elevation_max: 0.05 
hole_prob: 0.0 
hole_size_max: 7
